beacon_node/beacon_chain/tests/store_tests.rs

```diff
@@ -2547,6 +2548,25 @@ async fn weak_subjectivity_sync_test(slots: Vec<Slot>, checkpoint_slot: Slot) {
         }
     }
 
+    // Corrupt the signature on the 1st block to ensure that the backfill processor is checking
+    // signatures correctly. This is a regression test for XXX.
```

Comment:
> `XXX` 

Is this a placeholder for this PR number?

Reply:
ooh yeah, will fix